### PR TITLE
About page

### PR DESCRIPTION
**Pull request by bradymiller**
Terry added an about page. Almost done.

### CODE REVIEW

File: interface/login/login.php [hunk 1]
```diff
@@ -210,8 +210,8 @@ function imsubmitted() {
 
 </div>
 <div style="clear: both;"> </div>
+<a  href="../../acknowledge_license_cert.html" target="main"><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></a>
```
Comment (bradymiller) [inline interface/login/login.php]: This line should be below the version div below to keep the correct formatting.

File: interface/main/main_title.php [hunk 2]
```diff
@@ -117,8 +124,8 @@ function showhideMenu() {
 	<table cellspacing="0" cellpadding="1" style="margin:0px 3px 0px 0px;"><tr>
 		<td align="right" class="text" style="vertical-align:text-bottom;"><a href='main_title.php' onclick="javascript:parent.left_nav.goHome();return false;" ><?php xl('Home','e'); ?></a>
 		&nbsp;|&nbsp;
-		<a href="http://open-emr.org/wiki/index.php/OpenEMR_4.2.1_Users_Guide" target="_blank" id="help_link" >
-			<?php xl('Manual','e'); ?></a>&nbsp;</td>
+        <td align="right" class="text" style="vertical-align:text-bottom;"><a  href=""  onclick="return bpopup()" >
```
Comment (bradymiller) [inline interface/main/main_title.php]: Try this instead:
``````
<a  href=""  onclick="return bpopup()" >

File: interface/main/about_page.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+require_once("../globals.php");
+?>
+ <html>
+  <head>
+  <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+ </head>
+  <body class="body_top">
+    <div style="text-align: center;">
+    <table>
+    <span class=text><?php  echo xlt('About OpenEMR'); ?> </span><br><br>
```
Comment (bradymiller) [inline interface/main/about_page.php]: Change this to class="title"

File: interface/main/about_page.php [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+?>
+ <html>
+  <head>
+  <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+ </head>
+  <body class="body_top">
+    <div style="text-align: center;">
+    <table>
+    <span class=text><?php  echo xlt('About OpenEMR'); ?> </span><br><br>  
+	<span class=text><?php  echo xlt('Version Number'); ?>: <?php echo "v".text($openemr_version) ?></span><br><br>
```
Comment (bradymiller) [inline interface/main/about_page.php]: put quotes around the text:
``````
class="text">

File: interface/main/about_page.php [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    <?php if (!empty($GLOBALS['support_phone_number'])) { ?>
+   	  <span class=text><?php  echo xlt('Support Phone Number'); ?>: <?php echo $GLOBALS['support_phone_number'] ?></span>
```
Comment (bradymiller) [inline interface/main/about_page.php]: put quotes around the text

File: interface/main/about_page.php [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+$sanitize_all_escapes=true;
+  
+require_once("../globals.php");
+?>
+ <html>
+  <head>
+  <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+ </head>
+  <body class="body_top">
+    <div style="text-align: center;">
```
Comment (bradymiller) [inline interface/main/about_page.php]: This div is working and the About OpenEMR and Version are getting centered correctly.

File: interface/main/about_page.php [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    <table>
+    <span class=text><?php  echo xlt('About OpenEMR'); ?> </span><br><br>  
+	<span class=text><?php  echo xlt('Version Number'); ?>: <?php echo "v".text($openemr_version) ?></span><br><br>
+    <?php if (!empty($GLOBALS['support_phone_number'])) { ?>
+   	  <span class=text><?php  echo xlt('Support Phone Number'); ?>: <?php echo $GLOBALS['support_phone_number'] ?></span>
+    <?php } ?>
+
+
+   <div style="text-align: center;">
```
Comment (bradymiller) [inline interface/main/about_page.php]: Remove this div (you already did this above)

File: interface/main/about_page.php [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+   <br><a href="http://open-emr.org/wiki/index.php/OpenEMR_4.2.1_Users_Guide" target="_blank" id="support_link" class='css_button' onClick="top.restoreSession()"><span><?php echo xlt('User Manual'); ?></span></a>
```
Comment (bradymiller) [inline interface/main/about_page.php]: Lets put a \<br\> at the end of this line to see if it looks better.

File: interface/main/about_page.php (hunk 7 above)
Comment (bradymiller) [inline interface/main/about_page.php]: Guessing that removing this will get the stuff below to center.

File: interface/main/about_page.php [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    <?php if (!empty($GLOBALS['online_support_link'])) { ?>
+       <a href='<?php echo $GLOBALS["online_support_link"]; ?>' target="_blank" id="support_link" class='css_button' onClick="top.restoreSession()"><span><?php echo xlt('Online Support'); ?></span></a>
+    <?php } ?>
+   <br><br><a href="../../acknowledge_license_cert.html" target="_blank" id="support_link" class='css_button' onClick="top.restoreSession()"><span><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></span></a>
+   </div>
```
Comment (bradymiller) [inline interface/main/about_page.php]: remove this div tag

**Comment by bradymiller** [comment]
second review complete

File: interface/main/about_page.php [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+  
+require_once("../globals.php");
+?>
+ <html>
+  <head>
+  <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+ </head>
+  <body class="body_top">
+    <div style="text-align: center;">
+    <table>
```
Comment (bradymiller) [inline interface/main/about_page.php]: remove the table tag

File: interface/main/about_page.php [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+
+   <div style="text-align: center;">
+   <br><a href="http://open-emr.org/wiki/index.php/OpenEMR_4.2.1_Users_Guide" target="_blank" id="support_link" class='css_button' onClick="top.restoreSession()"><span><?php echo xlt('User Manual'); ?></span></a>
+    <?php if (!empty($GLOBALS['online_support_link'])) { ?>
+       <a href='<?php echo $GLOBALS["online_support_link"]; ?>' target="_blank" id="support_link" class='css_button' onClick="top.restoreSession()"><span><?php echo xlt('Online Support'); ?></span></a>
+    <?php } ?>
+   <br><br><a href="../../acknowledge_license_cert.html" target="_blank" id="support_link" class='css_button' onClick="top.restoreSession()"><span><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></span></a>
+   </div>
+    </table>
```
Comment (bradymiller) [inline interface/main/about_page.php]: remove the table tag

File: interface/login/login.php (hunk 1 above)
Comment (teryhill) [inline interface/login/login.php]: Done

File: interface/main/main_title.php (hunk 2 above)
Comment (teryhill) [inline interface/main/main_title.php]: Done

File: interface/main/about_page.php (hunk 6 above)
Comment (teryhill) [inline interface/main/about_page.php]: Ok

File: interface/main/about_page.php (hunk 10 above)
Comment (teryhill) [inline interface/main/about_page.php]: removed

File: interface/main/about_page.php (hunk 3 above)
Comment (teryhill) [inline interface/main/about_page.php]: Done

File: interface/main/about_page.php (hunk 4 above)
Comment (teryhill) [inline interface/main/about_page.php]: Done

File: interface/main/about_page.php (hunk 5 above)
Comment (teryhill) [inline interface/main/about_page.php]: Done

File: interface/main/about_page.php (hunk 7 above)
Comment (teryhill) [inline interface/main/about_page.php]: Did not center

File: interface/main/about_page.php (hunk 8 above)
Comment (teryhill) [inline interface/main/about_page.php]: looks worse

File: interface/main/about_page.php (hunk 9 above)
Comment (teryhill) [inline interface/main/about_page.php]: removed

File: interface/main/about_page.php (hunk 11 above)
Comment (teryhill) [inline interface/main/about_page.php]: removed

**Comment by bradymiller** [comment]
Hi,
I just committed this code to the codebase. Thanks for the contribution!
-brady

**Pull request closed without merging**